common/tvm_utility/test/yolo_v2_tiny/main.cpp

```diff
@@ -247,7 +247,9 @@ TEST(PipelineExamples, SimplePipeline)
   using PostPT = PostProcessorYoloV2Tiny;
 
   PrePT PreP{config};
-  IET IE{config, "tvm_utility"};
+  std::string home_dir = getenv("HOME");
```

Comment:
If the text fixtures are required to run this, they should be included in this repository, otherwise tests could not be reliably run (e.g. a user may provide different artifacts than what the tests are meant to run with). This is the same scenario as adding the JPEG image, which is also a test fixture.

Reply:
I'm not sure that it is such a big issue in current situation. Here we are talking about default case. If user will provide different artifacts or put them in the wrong place he will have the error in the test and will be informed that something is not happening as it's expected to be.  

We can introduce some additional env variable and reference it during the test. 
We also can remove test from the package as all tensorrt based packages do not have tests.  

Or git-lfs is the only way for current stage. But as I understand previous discussions It wasn't recognized as appropriate solution.